### PR TITLE
Add contributing guide

### LINKED ISSUE #511: Add contributing guide
# Summary

## Problem

It's not clear what we expect from contributors to the project.

### Impact

Contributing to the project is more difficult than it needs to be.

## Solution

Add a contributing guide.

### PR DESCRIPTION
**Pull request by bgins**
## Summary

This PR implements the following features:

* [x] Add contributing guide
* [x] Add code of conduct

This PR adds a contributing guide and code of conduct to set expectations around contributing and conduct in our project.

## Closing issues

Closes #511 

## After Merge
* [x] Does this change invalidate any docs or tutorials? No
* [x] Does this change require a release to be made? No

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+Maintainers have the right to remove and reject comments, threads, commits,
+code, documentation, pull requests, issues, and contributions not aligned with
+this Code of Conduct.
+
+## Scope
+
+This Code of Conduct applies within all project and community spaces, as well as
+in any public spaces where an individual representing the community is involved.
+This covers
```
Comment (avivash) [inline CODE_OF_CONDUCT.md:68]: nit
```suggestion
This covers:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+### Submitting a bug report
+
+We provide a template to give you a starting point when submitting a bug report. Please fill this template in with all relevant information, but feel free to delete any sections that do not apply.
+
+The most important information is describing the problem and how it impacts you. We also invite you to propose a solution, which could be a description of expected behavior.
+
+We very much appreciate it if you could include a [short, self-contained, correct example][sscce] that demonstrates the issue.
+
+## Pull requests
```
Comment (avivash) [inline CONTRIBUTING.md:36]: I wonder if we should mention something about how contributors need to make sure to sign their commits?(assuming we still have that setup?)

**Review by avivash — APPROVED** [review]
Couple small comments, but I think this looks great!

File: CONTRIBUTING.md (hunk 2 above)
Comment (bgins) [reply CONTRIBUTING.md:36]: Yes! That's a great point, we should include that. I'll add something. 👌

Comment (bgins) [reply CONTRIBUTING.md:36]: Alright, added this note:

https://github.com/oddsdk/ts-odd/blob/5c1664e1d8a2abc38eddab1b01cb77034ab07751/CONTRIBUTING.md?plain=1#L42

Comment (avivash) [reply CONTRIBUTING.md:36]: nice, looks great!

**Review by hugomrdias — APPROVED** [review]
LGTM

**Review by icidasset — APPROVED** [review]

**Pull request merged**